testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -198,6 +198,7 @@
     "MMTF_NOCRYST", # File with meaningless CRYST1 record (Issue #2679, PR #2685)
     "FHIAIMS", # to test FHIAIMS coordinate files
     "SDF_molecule"  # MDL SDFile for rdkit
```

Comment:
```suggestion
    "SDF_molecule",  # MDL SDFile for rdkit
```

This missing comma is just breaking all the tests :-)